lib/new_relic/agent/log_event_aggregator.rb

```diff
@@ -151,6 +161,8 @@ def reset!
           @seen = 0
           @seen_by_severity.clear
         end
+
+        attributes.reset!
```

Comment:
When is `reset!` called for this class? Are we sure that we would also want to reset the attributes that were set by the customer for future logs if this happens? 
If this is only a test thing, then it totally makes sense, but if we also do this when we reset the connection with the collector, i'm not sure.

Reply:
Removed in b83c43b20d62e9d6a0341aa0199c53825c409bfa